db-init/src/main/resources/database/migrations/V1.2__BIE_Contention_Event_Recreate_Table.sql

```diff
@@ -0,0 +1,19 @@
+-- Drop the old table
+DROP TABLE IF EXISTS "bie_contention_event";
+
+-- Create the new table
+CREATE TABLE IF NOT EXISTS "bie_contention_event"
```

Comment:
Since you're using the same table name, why not remove the 2 undesired columns (`event` and `event_details`) and add the new columns?

Reply:
It's much cleaner, especially with data already existing that will be inconsistent to reset the table. There is not much cost for it.